src/components/dashboard/FaceVerification/sdk/EnrollmentProcessor.web.js

```diff
@@ -231,25 +241,119 @@ export class EnrollmentProcessor {
 
   /**
    * generates session ID and starts session
-   * enroll call proxioes here - just for keep non-async
+   * enroll call proxies here - just for keep non-async
    * interface with onComplete callback designed by Zoom
    * @private
    */
   async _startEnrollmentSession() {
-    const { subscriber } = this
+    const { subscriber, _waitForSDKUIElementVisible } = this
 
     try {
       // trying to retrieve session ID from Zoom server
       const sessionId = await api.issueSessionToken()
 
-      // if we've got it - strting enrollment session
-      new FaceTecSession(this, sessionId)
-
       // notifying subscriber that UI is ready
-      subscriber.onUIReady()
+      _waitForSDKUIElementVisible('DOM_FT_getReadyActionButton', () => subscriber.onUIReady())
+
+      // If device orientation changed before FV session initialized and ready screen shown,
+      // SDK shows camera permissions popup (even the built-in camera screen is disabled in UICustomiuzation).
+      // If press OK in this dialog, FV session then crashes with white screen
+      // Removing Zoom's UI from DOM removes blurred background but it's impossible to start session again
+      // Only app reloading solves the issue and allows to retry FV attempt.
+      // So, in that case we're redirecting app to the corresponding FV error scrren with full page reload
```

Comment:
`      // So, in that case we're redirecting app to the corresponding FV error screen with full page reload`